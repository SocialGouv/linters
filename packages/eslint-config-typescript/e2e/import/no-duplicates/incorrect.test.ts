import {
  fixtures,
  fixtureToTestCase,
  ruleTest,
  TIMEOUT,
} from "e2e/rule_tester";
import { basename } from "path";

describe(`import/${basename(__dirname)} error`, () => {
  test.each(fixtures(__dirname, /^incorrect$/).map(fixtureToTestCase))(
    "should invalidate : %s",
    (_, filePath) => ruleTest(filePath),
    TIMEOUT
  );
});
