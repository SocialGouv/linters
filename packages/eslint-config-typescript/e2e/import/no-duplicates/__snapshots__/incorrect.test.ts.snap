// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import/no-duplicates error should invalidate : index.ts 1`] = `
Object {
  "errorCount": 0,
  "fixableErrorCount": 0,
  "fixableWarningCount": 0,
  "results": Array [
    Object {
      "errorCount": 0,
      "filePath": StringContaining "e2e/import/no-duplicates/__fixtures__/incorrect/index.ts",
      "fixableErrorCount": 0,
      "fixableWarningCount": 0,
      "messages": Array [],
      "warningCount": 0,
    },
  ],
  "usedDeprecatedRules": Array [],
  "warningCount": 0,
}
`;
